<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSystemcheck</class>
 <widget class="QDialog" name="DlgSystemcheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>维护检测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/image/logo.png</normaloff>:/images/image/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_syscheck">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>470</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_io">
    <attribute name="title">
     <string>IO输入</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_history">
    <attribute name="title">
     <string>历史信息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>85</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
